### DISCUSSION TITLE
Workflows -- use case

### DISCUSSION THREAD

**Discussion by dturton** [taskforcesh/bullmq#510] [Q&A]
I know this may not be the ideal place to ask for help on this but I thought I would give it a try. Example use case, e-commerce order processing. I saw the new feature for Flow but I am not sure if it solves my use case. My ideal scenario is first an initial job is created then based on its return value then you decide what the next step should be. For example, the order is placed, if the order is above $100 then do these additional jobs and pass the return value to them. I would like to use the Flow feature but I can't find a solution on how to use the return value of a job to decide which task to perform or not to perform next.  The reason I need it decoupled as some of the tasks require calling a third-party API, if it fails I want to retry just that one step. I hope it makes sense.

**Comment by manast** [comment]
you do not need flows to achieve this feature, just use two more queues and add a job to one queue or the other from your "parent" job.

**Reply by dturton** [reply]
Thank you for moving this to Discussions.

Any chance you have a quick snippet for this? By "parent", you mean the first job that then calls the second? A quick example would be very helpful if you can. It's still important for me to be able to track when all the child jobs complete guaranteeing that the order was processed successfully.
